src/styles/blocks/card.scss

```diff
@@ -0,0 +1,84 @@
+.card {
+  font-family: Roboto, Helvetica, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: $color-secondary;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  width: 200px;
```

Comment:
css properties order should be like that
http://codeguide.academy/html-css.html#css-order

fix all such cases